assets/stylesheets/screens/home.scss

```diff
@@ -0,0 +1,16 @@
+.home {
```

Comment:
Bind it to the `body` tag to avoid with conflicts with a component named `home` for instance 💥 :

```suggestion
body.home {
```

make sure this is done for all screen stylesheets.

Reply:
Fixed in daa3a488ad94ff88a96a1ebc77bb2af1e75b3456